### ISSUE TITLE
The default setting only supports L2 loss as reward

### ISSUE THREAD

**Issue by Vandermode** [hzwer/ICCV2019-LearningToPaint#29]
Hi, it seems the default training setting only supports L2 loss as reward, modifying it to wgan loss is slightly non-trivial.

**Comment by Vandermode** [comment]
Ok.. I understand... the rewards given by ```env``` are not used in the ```update_policy```..

**Comment by yael-vinker** [comment]
regarding this topic, it seems like the L2 used as reward from the environment is used during the training.
could you please explain how it is being used and what is the different in terms of the learning process between the L2 reward from the env and the wgan reward in the "update_policy".
Does the model learns to optimise over L2 as well?
thanks.

**Comment by hzwer** [comment]
In the baseline, l2 reward should not participate in model training. There is an optimized loss function for reinforcement learning https://github.com/megvii-research/ICCV2019-LearningToPaint/blob/24e317ba1d7c88435677fc77cb2ded6d03b2a914/baseline/DRL/ddpg.py#L138, which is used to drive Q function learning.
In other words, wgan or l2 reward determines the single-step reward of the model. In the baseline experiment, only wgan reward was used instead of the l2 reward of the environment.
